ecosystem/sep-0038.md

```diff
@@ -0,0 +1,176 @@
+
+This endpoint can be used to request a [firm](https://www.investopedia.com/terms/f/firmquote.asp) quote for a Stellar asset and currency pair. 
+
+In contrast with the `GET price(s)` endpoints, the amount requested must be held in reserve and not used in calculations of subsequent quotes until the expiration provided in the response.
+
+#### Protecting Against Bad Actors
+
+To protect against bad actor clients reserving all available captial without following through with the requested trades, servers should only accept requests for which the following is true:
+
+1. The Stellar account authenticated via SEP-10 is owned by a known and trusted entity, or
+2. The owner of the Stellar account has been properly KYC'ed, either via [SEP-12](sep-0012.md) or some other mechanism.
+
+Entities or individuals that misuse the service can be placed on a denylist that should be checked before processing requests for quotes.
+
+#### Transaction Fees
+
+It should be expected that the provided quote includes a margin for the provider as a service fee, and this margin will vary depending on the directional flow of funds.
+
+If the client requests some amount of off-chain currency for providing some amount of a Stellar asset, the client will submit a Stellar transaction delievering funds to the anchor before the expiration included in the response, paying a fee as determined by state of the Stellar network. 
+
+In the reverse scenario, the anchor will submit a Stellar transaction to deliver funds to the client as long as the client delivered off-chain funds to the anchor before the expiration. In this case, the anchor will likely increase their margin to cover the cost of submitting the transaction.
+
+This specification could be extended in the future to include functionality where the anchor collects a transaction signature  from the client prior to submitting the transaction, enabling the anchor to have the client pay the network fees and decrease their margin. This feature would increase the technical complexity of the specification and hence will only be added if there is enough support from the ecosystem.
+
+#### Request
+
+Name | Type | Description
+-----|------|------------
+`base` | string | Same as the definition of `base` in `GET /price`.
+`counter` | string | Same as the definition of `counter` in `GET /price`.
+`base_amount` | string | Same as the definition of `base_amount` in `GET /price`.
+`delivery_method` | string | (optional) One of the `name` values specified by the `delievery_methods` array for the relevant currency returned from `GET /info`. If the array is empty, this parameter is not required.
+
+#### Response
+
+Name | Type | Description
+-----|------|------------
+`id` | string | The unique identifier for the quote to be used in other Stellar Ecosystem Proposals (SEPs).
+`expiration` | [UTC ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) string | The date and time by which the anchor must receive funds from the client.
```

Comment:
💡 I'm inclined to suggest naming the `expiration` field `expires_at`. The use of `_at` is reasonably common in APIs to signal timestamps.
```suggestion
`expires_at` | [UTC ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) string | The date and time by which the anchor must receive funds from the client.
```